src/promptflow/promptflow/contracts/run_info.py

```diff
@@ -20,16 +22,65 @@ class Status(Enum):
 
     @staticmethod
     def is_terminated(status):
+        """Check if a given status is terminated.
+
+        :param status: The status to be checked
+        :type status: str or :class:`Status`
+        :return: True if the status is terminated, False otherwise
+        :rtype: bool
+        """
         if isinstance(status, Status):
             status = status.value
         return status in {s.value for s in {Status.Completed, Status.Failed, Status.Bypassed, Status.Canceled}}
 
 
 @dataclass
 class RunInfo:
-    node: str  # Node name
-    flow_run_id: str  # This is equal to root_run_id
-    run_id: str  # flow_run_id:step_run_id
+    """A dataclass representing the run information.
+
+    :param node: Node name
+    :type node: str
+    :param flow_run_id: This is equal to root_run_id
+    :type flow_run_id: str
+    :param run_id: flow_run_id:step_run_id
```

Comment:
Update this to a valid English sentence.

Reply:
Changed to "The id of the run, which equals flow_run_id:step_run_id"